### DISCUSSION TITLE
Agentflow and chatflow version control process

### DISCUSSION THREAD

**Discussion by mapachekurt** [FlowiseAI/Flowise#3155] [Q&A]
I'd like to know if anyone has developed an (ideally) automated way to accomplish a rudimentary version control process. I know you can export and import flows, but only in the GUI, as even the latest API release doesn't have that capability. I don't see any version control threads in Q&A, which seems weird, but hopefully somebody is thinking along these lines.

**Comment by mdlmarkham** [comment]
+10 This is very important, we need to have version control with the ability to track changes, roll-back etc. I don't know if this is something that should be built in, or if it makes more sense to integrate with GitHub.
